src/Main.hs

```diff
@@ -18,7 +18,7 @@ import Control.Concurrent
 import Data.IORef
 
 import FuncTorrent.Bencode (decode)
-import FuncTorrent.Logger (initLogger, logMessage, logStop)
+import FuncTorrent.Logger (Log, initLogger, logMessage, logError, logStop)
```

Comment:
Just being explicit. Future proof! Makes it easy to grep. Small things like that.